addons/l10n_at/data/template/account.account-at.csv

```diff
@@ -1,198 +1,235 @@
+"chart_at_template_2290","Global valuation allowances on receivables from other long-term investees and investors","2290","False","asset_current","l10n_at.account_tag_external_code_2290","Pauschalwertberichtigungen zu Forderungen gegenüber Unternehmen, mit denen ein Beteiligungsverhältnis besteht"
+"chart_at_template_2300","Other receivables and other assets","2300","False","asset_current","l10n_at.account_tag_external_code_2300","Sonstige Forderungen und Vermögensgegenstände"
+"chart_at_template_2470","Unpaid called capital contributions","2470","False","asset_current","l10n_at.account_tag_external_code_2470","Eingeforderte, aber noch nicht eingezahlte Einlagen"
+"chart_at_template_2480","Specific valuation allowances on other receivables and other assets","2480","False","asset_current","l10n_at.account_tag_external_code_2480","Einzelwertberichtigungen zu sonstigen Forderungen und Vermögensgegenständen"
+"chart_at_template_2490","Global valuation allowances on other receivables and other assets","2490","False","asset_current","l10n_at.account_tag_external_code_2490","Pauschalwertberichtigungen zu sonstigen Forderungen und Vermögensgegenständen"
+"chart_at_template_2500","Input tax 20%","2500","False","asset_current","l10n_at.account_tag_external_code_2500","Vorsteuern 20%"
+"chart_at_template_2501","Input tax 10%","2501","False","asset_current","l10n_at.account_tag_external_code_2500","Vorsteuern 10%"
+"chart_at_template_2502","Input tax 13%","2502","False","asset_current","l10n_at.account_tag_external_code_2500","Vorsteuern 13%"
+"chart_at_template_2505","Other tax 13%","2505","False","asset_current","l10n_at.account_tag_external_code_2500","Sonstige Vorsteuern"
+"chart_at_template_2506","Input tax RC 20%","2506","False","asset_current","l10n_at.account_tag_external_code_2502","Vorsteuern RC 20%"
+"chart_at_template_2507","Input tax RC 10%","2507","False","asset_current","l10n_at.account_tag_external_code_2502","Vorsteuern RC 10%"
+"chart_at_template_2510","Input tax RC EU 20%","2510","False","asset_current","l10n_at.account_tag_external_code_2502","Vorsteuern RC EU 20%"
+"chart_at_template_2511","Input tax on intra-Community acquisitions 20%","2511","False","asset_current","l10n_at.account_tag_external_code_2501","Vorsteuern IGE 20%"
+"chart_at_template_2512","Input tax on intra-Community acquisitions 10%","2512","False","asset_current","l10n_at.account_tag_external_code_2501","Vorsteuern IGE 10%"
+"chart_at_template_2513","Input tax on intra-Community acquisitions 13%","2513","False","asset_current","l10n_at.account_tag_external_code_2501","Vorsteuern IGE 13%"
+"chart_at_template_2515","Input tax 20% (from import VAT)","2515","False","asset_current","l10n_at.account_tag_external_code_2509","Vorsteuern 20% (aus EUSt.)"
+"chart_at_template_2610","Shares in affiliated companies","2610","False","asset_current","l10n_at.account_tag_external_code_2610","Anteile an verbundenen Unternehmen"
```

Comment:
Name is present twice

Reply:
2610 and 0800 both also have the same name in the SAF-T COA. They existed previously like that too.